components/TabsContainer.vue

```diff
@@ -2,7 +2,7 @@
   <b-card class="main-card" no-body>
     <b-tabs class="main-tabs" card pills align="center" lazy>
       <b-tab title="Summary" active>
-        <div v-if="attemptsExist">
+        <div v-if="parsedSplits.Run.AttemptHistory">
```

Comment:
Is this change intended ? You updated `attemptsExist` but you're not using it anymore

Reply:
when filters are active, attemptsExist checks that AttemptHistory exists and that there are attempts in the filtered list, but since none of the filters affect the summary tab, the v-if is just AttemptHistory so that it only gets disabled if there no attempts in the file and not when all are filtered out
there's probably a prettier way to do it but this seemed the most simple

also attemptsExist is used for all of the other tabs, but that's probably not visible in the commit

Reply:
Fair, fair, thanks for explanation, you're right